scoresheets/CleanTable.tex

```diff
@@ -2,18 +2,18 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[5]{60}{Picking tableware and cutlery for transportation to the dishwasher}
-	\scoreitem[5]{180}{Placing the tableware and cutlery inside the dishwasher}
+	\scoreitem[5]{120}{Picking tableware and cutlery for transportation to the dishwasher}
+	\scoreitem[5]{80}{Placing the tableware and cutlery inside the dishwasher}
+	\scoreitem[5]{80}{Placing an item in the correct place in the dishwasher}
 
 	\scoreheading{Bonus Rewards}
 	\scoreitem{200}{Opening the dishwasher door}
 	\scoreitem{100}{Pulling out the dishwasher rack}
-	\scoreitem[5]{40}{Placing an item correctly in the dishwasher}
-	\scoreitem{300}{Placing the dishwasher tab inside the dishwasher}
+	\scoreitem{300}{Picking and Placing the dishwasher tab inside the dishwasher}
 
 	\scoreheading{Deus Ex Machina Penalties}
 	\penaltyitem[5]{25}{Pointing at an object or telling the robot where an object is}
-	\penaltyitem[5]{60}{Handing an object over to the robot}
+	\penaltyitem[5]{0}{Handing an object over to the robot (no points are awarded for pickup)}
```

Comment:
It goes in accordance to the comment  above. Is is hard to score partial deliveries. The reduction should be applied only if the take and delivery is performed.